mapper/mapper.go

```diff
@@ -122,9 +124,12 @@ func CreateFilterOverview(dimensions []filter.ModelDimension, filter filter.Mode
 func CreateListSelectorPage(name string, selectedValues []filter.DimensionOption, allValues dataset.Options, filter filter.Model, dst dataset.Model, datasetID, releaseDate string) listSelector.Page {
 	var p listSelector.Page
 
+	log.Debug("mapping api response models to list selector page model", log.Data{"filterID": filter.FilterID, "datasetID": datasetID})
```

Comment:
Would name be useful here? gives context for which specific area inside a dataset is causing issues